packages/core/lib/test.js

```diff
@@ -170,14 +170,6 @@ const Test = {
     // Propagate --bail option to mocha
     mochaConfig.bail = config.bail;
 
-    // If the command line overrides color usage, use that.
-    if (config.colors != null) mochaConfig.useColors = config.colors;
-
-    // Default to true if configuration isn't set anywhere.
-    if (mochaConfig.useColors == null) {
-      mochaConfig.useColors = true;
```

Comment:
Looks like `useColors` got removed in `mocha` `v8` https://github.com/mochajs/mocha/pull/4178,
and is now instead `color` https://mochajs.org/api/mocha.js.html#line696.

Requesting changes to keep these lines and replace `mochaConfig.useColors` for `mochaConfig.color`.

Reply:
Yep that sounds good.